app/services/analytics_events.rb

```diff
@@ -565,6 +565,7 @@ def idv_verify_in_person_troubleshooting_option_clicked(flow_path:, **extra)
     )
   end
 
+  # @param [Hash] variant The IPP CTA Variant that was shown to the user
```

Comment:
variant sounds more like a String or Symbol than a hash?
```suggestion
  # @param [String] variant The IPP CTA Variant that was shown to the user
```

Reply:
Thanks Zach. I had gotten mixed up in how the arguments were unpacked and passed through!